torch/csrc/autograd/profiler_kineto.cpp

```diff
@@ -151,6 +152,7 @@ struct KinetoThreadLocalState : public ProfilerThreadLocalState {
         act.addMetadata("Device Type", std::to_string((int8_t)device.type()));
         act.addMetadata("Device Id", std::to_string(device.index()));
         act.addMetadata("Bytes", std::to_string(alloc_size));
+        act.addMetadata("Reserved Bytes", std::to_string(reserved_size));
```

Comment:
forget add Allocated_Bytes?

Reply:
Nah, `alloc_size` is not diff anymore. Will change it later.

Reply:
Should be good now.